<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TotpSetupDialog</class>
 <widget class="QDialog" name="TotpSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup TOTP</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Key:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="seedEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioDefault">
        <property name="text">
         <string>Default RFC 6238 token settings</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">settingsButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioSteam">
        <property name="text">
         <string>Steam token settings</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">settingsButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioCustom">
        <property name="text">
         <string>Use custom settings</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">settingsButtonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="customGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Custom Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="rowWrapPolicy">
       <enum>QFormLayout::DontWrapRows</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="stepLabel">
        <property name="text">
         <string>Time step:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="stepSpinBox">
        <property name="suffix">
         <string comment="Seconds"> sec</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="digitsLabel">
        <property name="text">
         <string>Code size:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="radio6Digits">
        <property name="text">
         <string>6 digits</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="radio7Digits">
        <property name="text">
         <string>7 digits</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="radio8Digits">
        <property name="text">
         <string>8 digits</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>seedEdit</tabstop>
  <tabstop>radioDefault</tabstop>
  <tabstop>radioSteam</tabstop>
  <tabstop>radioCustom</tabstop>
  <tabstop>stepSpinBox</tabstop>
  <tabstop>radio6Digits</tabstop>
  <tabstop>radio7Digits</tabstop>
  <tabstop>radio8Digits</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="settingsButtonGroup"/>
 </buttongroups>
</ui>
